.fossa.yml

```diff
@@ -691,6 +691,12 @@ targets:
     - type: gradle
       path: ./
       target: ':instrumentation:mongo:mongo-async-3.3:javaagent'
+    - type: gradle
+      path: ./
+      target: ':instrumentation:nats:nats-2.21:javaagent'
```

Comment:
since it passes for 2.17.2+, should I change 2.21 to 2.17.2 everywhere? (library included)